services/mlmodel/tflitecpu/tflite_cpu.go

```diff
@@ -0,0 +1,249 @@
+// Package tflitecpu is meant to implement a tfliteCPU in the ML model service
```

Comment:
Update this package string - it runs tflite model files on host's CPU

Reply:
Done